libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route2/ReplayRouteOptions.kt

```diff
@@ -52,6 +54,9 @@ class ReplayRouteOptions(
 
         /**
          * Max speed the driver will drive on straight-aways
+         *
+         * @param maxSpeedMps the driver will drive on straight-aways
```

Comment:
Are these duplicate comments meaningful? Delete the description or only put the units on the @param